modules/installation-gcp-permissions.adoc

```diff
@@ -21,6 +21,7 @@ When you attach the `Owner` role to the service account that you create, you gra
 
 .Required roles for the installation program
 * Compute Admin
+* IAM Role Administrator
```

Comment:
**Install (==Mint)**

Do the ones that were already in here apply, or should they be removed to leave only "IAM Role Administrator"?

Reply:
Yes, we are adding IAM Role Administrator the the existing list.